src/Pagination/README.md

```diff
@@ -94,3 +96,38 @@ notes: |
   }}
 />
 ```
+
+### Theme variables (SCSS)
```

Comment:
Theme variables no longer need to be explicitly added to the README since the PR to automatically add them was recently merged ([PR](https://github.com/edx/paragon/pull/931)). Refer to the examples there. Apologies for the delayed review.

Reply:
Yes, I am going to remove `Theme variables` everywhere from .md